### DISCUSSION TITLE
如果吧ya-webadb拆分开

### DISCUSSION THREAD

**Discussion by yaomoshen520** [yume-chan/ya-webadb#459] [Q&A]
如何把这个项目成分开，比如我手动的通过adb 启动 scrcpy-server，
adb forward tcp:5000 localabstract:scrcpy

adb push ./scrcpy-server /data/local/tmp/scrcpy-server.jar

adb shell CLASSPATH=/data/local/tmp/scrcpy-server.jar app_process / com.genymobile.scrcpy.Server 1.22 log_level=info bit_rate=8000000 tunnel_forward=true lock_video_orientation=0 send_frame_meta=false encoder_name=OMX.qcom.video.encoder.avc
然后如何把这个端口数据 显示在 ya-webadb中，意思就是把这个项目当做一个Client端展示和控制

**Comment by yume-chan** [comment]
My Scrcpy library can be used like that, however it doesn't have any UI, you must create it yourself.

See the README (https://github.com/yume-chan/ya-webadb/tree/main/libraries/scrcpy#using-other-transportation) and https://github.com/yume-chan/ya-webadb/issues/455#issuecomment-1282207777
